package.json

```
@@ -47,6 +47,7 @@
   },
   "homepage": "https://github.com/benmosher/eslint-plugin-import",
   "devDependencies": {
+    "@eslint/import-test-order-redirect-scoped": "file:./tests/files/order-redirect-scoped",
     "@typescript-eslint/parser": "^1.5.0",
```

```suggestion
     "@typescript-eslint/parser": "1.10.3-alpha.13",
```
Regarding the `@typescript-eslint/parser` bug, `@typescript-eslint/parser` releases every commit to master as a canary version. You can use it for now.